pkg/client/src/app/pages/applications/components/application-form/application-form.tsx

```diff
@@ -342,7 +345,7 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({
       name: formValues.name.trim(),
       description: formValues.description.trim(),
       comments: formValues.comments.trim(),
-      businessService: formValues.businessService
+      businessService: formValues?.businessService?.id
```

Comment:
This will result in `businessService` being `0` in the payload when none is selected, not sure if that is desired. I wonder if it's okay to switch the initial value back to null, and have this be
```
businessService: formValues?.businessService?.id || null
```

Reply:
Ah, makes sense ok.